cookiecutter/config.py

```diff
@@ -60,13 +60,8 @@ def get_config(config_path):
         )
 
     logger.debug('config_path is %s', config_path)
-    with open(config_path, encoding='utf-8') as file_handle:
-        try:
-            yaml_dict = poyo.parse_string(file_handle.read())
-        except poyo.exceptions.PoyoException as e:
-            raise InvalidConfiguration(
-                'Unable to parse YAML file {}. Error: {}'.format(config_path, e)
-            )
+    with open(config_path, 'r') as yaml_file:
```

Comment:
Please specify the encoding, this is needed to avoid issues on some platforms where `utf-8` is not the default.